tests/unit/utils/test_network.py

```diff
@@ -701,3 +701,50 @@ def test_generate_minion_id_with_long_hostname(self):
             # An exception is raised if unicode is passed to socket.getfqdn
             minion_id = network.generate_minion_id()
         assert minion_id != '', minion_id
+
+    def test_filter_by_networks_with_no_filter(self):
+        ips = ["10.0.123.200", "10.10.10.10"]
+        self.assertRaises(TypeError, network.filter_by_networks, ips)
```

Comment:
I've not used `with pytest.raises` here because I was not sure if it is ok to "`import pytest` or skip"¹ this test. Although it just came to my mind that if `assert a == b` requires `pytest`, it would make sense to skip these all tests if `pytest` is unavailable.

¹ like it is done [here](https://github.com/saltstack/salt/blob/master/tests/unit/utils/test_thin.py#L24)

Reply:
This try is two years old. I would assume pytest available.